site/content/configuration/health-checks.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Health Checks"
+draft: false
+weight: 400
+toc: true
+tags: [ "docs" ]
+docs: "DOCS-XXX"
+categories: ["configuration"]
+doctypes: ["task"]
+---
+
+## Overview
+To monitor the state of the NGINX Agent, there is a health endpoint in the REST API that can be used.
+
+## Configure REST API
+To enable the REST API, the following configuration is needed:
+```nginx configuration
+api:
+  host: 127.0.0.1
+  port: 8038
+```
```

Comment:
```suggestion
## Configure the REST API

To enable the REST API, use the following configuration:

```nginx configuration
api:
  host: 127.0.0.1
  port: 8038
```
```

Markdown: Blank lines between different types of content.
Avoid passive.